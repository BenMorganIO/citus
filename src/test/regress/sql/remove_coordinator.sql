-- removing coordinator from pg_dist_node should update pg_dist_colocation
SELECT master_remove_node('localhost', :master_port);
SELECT replicationfactor = 2 FROM pg_dist_colocation WHERE distributioncolumntype=0;
