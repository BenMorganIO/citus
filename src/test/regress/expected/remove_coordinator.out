-- removing coordinator from pg_dist_node should update pg_dist_colocation
SELECT master_remove_node('localhost', :master_port);
 master_remove_node 
--------------------
 
(1 row)

SELECT replicationfactor = 2 FROM pg_dist_colocation WHERE distributioncolumntype=0;
 ?column? 
----------
 t
(1 row)

